test/e2e/cypress/e2e/clusters_overview.cy.js

```diff
@@ -79,6 +82,7 @@ context('Clusters Overview', () => {
       });
     });
 
+    // eslint-disable-next-line mocha/no-skipped-tests
```

Comment:
Tests should not be skipped because it's the equivalent of dead or commented-out code.

This is enforced by the `mocha/no-skipped-tests` rule. However, I decided to disable the rule for existing implementation.